backtesting/lib.py

```diff
@@ -452,6 +452,43 @@ def next(self):
                                self.data.Close[index] + self.__atr[index] * self.__n_atr)
 
 
+class PercentageTrailingStrategy(Strategy):
+    """
+    A strategy with automatic trailing stop-loss, trailing the current
+    price at distance of some percentage. Call
+    `PercentageTrailingStrategy.set_trailing_sl()` to set said percentage
+    (`5` by default). See [tutorials] for usage examples.
+
+    [tutorials]: index.html#tutorials
+
+    Remember to call `super().init()` and `super().next()` in your
+    overridden methods.
+    """
+    _sl_percent = 5.
+
+    def init(self):
+        super().init()
+
+    def set_trailing_sl(self, percentage: float = 5):
+        assert percentage > 0, "percentage must be greater than 0"
+        """
+        Sets the future trailing stop-loss as some (`percentage`)
+        percentage away from the current price.
+        """
+        self._sl_percent = percentage
```

Comment:
`position.pl_pct ` is in 0.XX. 
To avoid confusion, you may change _sl_percent to _sl_pct, and change the value to percentage/100 ?